.github/workflows/continuous-integration.yml

```diff
@@ -125,11 +92,10 @@ jobs:
       - name: Test with pytest
         run: pytest --cov=./ --cov-report=xml
       - name: Upload coverage
-        uses: codecov/codecov-action@v1
+        uses: codecov/codecov-action@v3
 
   test-conda:
```

Comment:
Is there a difference from python tests?

Reply:
Well, yes and no. We are running the same python test suite, but in a different context: conda env rather than virtual env, and there we also test linux/macos/windows. I think the multi-os dimension is what I care most about here.

Reply:
How about ci-build-wheel then? It is more automated and more portable.

Reply:
I have created a follow-up at https://github.com/mwouts/jupytext/issues/1086